src/main/java/com/sngular/kloadgen/sampler/KafkaConsumerSampler.java

```diff
@@ -48,6 +48,7 @@ public final void setupTest(final JavaSamplerContext context) {
 
   public final Properties properties(final JavaSamplerContext context) {
     final var props = SamplerUtil.setupCommonConsumerProperties(context);
+    //todo: obtener unas properties u otras en funcion de si el schema registry es de Confluent o de Apicurio
```

Comment:
I think if the project document is in English, don't make comments in Spanish.

Reply:
That comment is going to remove